Web UI/CognitiveSearch.UI/wwwroot/js/results.js

```diff
@@ -319,12 +323,17 @@ function UpdateResults(data) {
             name = "";
         }
 
+        var thumbnail_size_ext = "_300xN.jpg";
+        var thumbnail_path = "https://kmogstorage.blob.core.windows.net/kmdocument/THUMBNAILS/";
```

Comment:
are we sure hardcoding the blob path is the correct implementation here?
what happens if the container is moved, or we want to fetch the images from a different location?